web/docs/tql2/operators/measure.md

```diff
@@ -12,16 +12,16 @@ The `measure` operator yields metrics for each received batch of events or bytes
 using the following schema, respectively:
```

Comment:
Actually, thinking a bit more about this I'm not sure if it's a good idea to make the parameter *this* hidden. It would be good if a future frontend developer could figure out what it does without reading the source code of the measure operator.

Maybe we could document it here with a disclaimer like

```
For internal use only. This parameter makes the measure operator only output the definition the first time an event with a given schema is encountered. This is intended for internal use by the platform and is subject to notice change without any notice or advance warning.
```

Reply:
We could always expose that later on, but it's important to get this into the release now. I think the debate on whether this is a hidden argument or a `platform::create_schema_tree` operator of its own is not yet fully settled, and before then I wouldn't want to document it.